ci/cloudbuild/builds/asan.sh

```diff
@@ -29,3 +30,5 @@ bazel test "${args[@]}" --test_tag_filters=-integration-test ...
 
 mapfile -t integration_args < <(integration::bazel_args)
 integration::bazel_with_emulators test "${args[@]}" "${integration_args[@]}"
+
+conformance::bazel_with_proxies
```

Comment:
...and appending `"${args[@]}" "${integration_args[@]}"` here.

Reply:
I only appended `"${args[@]}"`, as `integration_args` is not relevant to the conformance test (e.g. it sets project/instance values)

Reply:
Good call. Apparently `bazel build/run` do not care about `--test_env=...`s